examples/ibm-sdsaas/README.md

```diff
@@ -0,0 +1,97 @@
+# Examples for sdsaas
+
+These examples illustrate how to use the resources and data sources associated with sdsaas.
+
+The following resources are supported:
+* ibm_sds_volume
+* ibm_sds_host
+
+## Usage
+
```

Comment:
This might be a good place to mention they need to set the IBMCLOUD_SDS_ENDPOINT env var?

Reply:
Removed